src/intl/en/page-layer-2-learn.json

```diff
@@ -0,0 +1,55 @@
+{
+  "page-layer-2-learn-meta-title": "What is layer 2",
+  "page-layer-2-learn-title": "What is layer 2",
+  "page-layer-2-learn-description": "Scaling Ethereum for mass adoption",
+  "page-layer-2-learn-button-1-label": "What is layer 2",
+  "page-layer-2-learn-button-2-label": "Use layer 2",
+  "page-layer-2-learn-what-is-layer-2-title": "What is layer 2?",
```

Comment:
Seeing 4 instances of "What is layer 2"... one of which has a question mark

1. Should we have a question mark at the end of this string? (I think we should)
2. Do we need to repeat this key, or can we simplify to just have one instance of "What is layer 2?"